### PR TITLE
VACMS-10982 - Prototype updates: Fixing broken links and typos

### PR DESCRIPTION
**Pull request by apisandipas**
## Description
Homepage prototype tweaks as outlined in department-of-veterans-affairs/va.gov-cms/issues/10982

## Testing done
Cross referencing changes as outline in the ticket. Eyeballing image adjustments in accordance with the prototype sketch file.

## Acceptance criteria
- [x]  Update Pathfinder image
- [x] Update `More VA news` link in News promo area
- [x] Update `Find a VA facility` link under Other search tools
- [x] Update/remove Departments link
- [x] Address typos and punctuation in benefit hubs
- [ ] Add icon needed for VA departments _- this has already been requested from Ryan_  (still awaiting this icon for changing)

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/site/stages/build/drupal/home.js [hunk 1]
```diff
@@ -103,33 +103,33 @@ function addHomeContent(contentData, files, metalsmith, buildOptions) {
...
       const vaDeptInfoHub = {
         entity: {
           entityId: '123456',
           fieldTitleIcon: '',
-          path: '#',
+          path: '',
           fieldHomePageHubLabel: 'VA department information',
           fieldTeaserText:
-            'Learn more about the VA departments that manage our benefits and health care programs',
+            'Learn more about the VA departments that manage rour benefits and health care programs.',
```
Comment (maxx1128) [inline src/site/stages/build/drupal/home.js:132]: "your," not "rour."

File: src/site/layouts/home-preview.drupal.liquid [hunk 2]
```diff
@@ -38,12 +38,10 @@
               </div>
               <div class="vads-l-col--12 medium-screen:vads-l-col--5">
                 <div id="email-update-signup-form" class="vads-u-margin-top--2p5 medium-screen:vads-u-margin-top--0">
-                  <label class="usa-sr-only">Sign Up for Email Updates</label>
-                  <input type="text" class="email-update-signup-form__input
+                  <input type="email" autocomplete="email" aria-label="Email Address" class="email-update-signup-form__input
```
Comment (maxx1128) [inline src/site/layouts/home-preview.drupal.liquid:41]: Should the `aria-label` here still be "Sign Up for Email Updates?" That makes the input a little more explicit that users are signing up for updates. Just "email address" is a bit vague.

**Review by maxx1128 — CHANGES_REQUESTED** [review]
Looks good, there are just two small things I noticed.

File: src/site/layouts/home-preview.drupal.liquid (hunk 2 above)
Comment (apisandipas) [reply src/site/layouts/home-preview.drupal.liquid:41]: This was a direct request from the Accessibility folks.

**Review by maxx1128 — APPROVED** [review]

**Pull request merged**